webapp/src/main/java/ar/edu/itba/paw/webapp/controllers/old/OldStudentController.java

```diff
@@ -0,0 +1,783 @@
+//		mav.addObject("subsection_courses", true); /* only different line from /inscription */
+//		mav.addObject("courses", courses);
+//		mav.addObject("docket", docket);
+//		return mav;
+//	}
+
+//	@RequestMapping(value = "/students/{docket}/courses/unenroll", method = RequestMethod.POST)
+//	public ModelAndView unenroll(@PathVariable final Integer docket,
+//								 @Valid @ModelAttribute("inscriptionForm") InscriptionForm inscriptionForm,
+//								 final BindingResult errors,
+//								 final RedirectAttributes redirectAttributes,
+//								 @ModelAttribute("user") UserSessionDetails loggedUser) {
+//
+//		if (!loggedUser.hasAuthority("DELETE_INSCRIPTION")
+//				|| (loggedUser.getId() != docket && !loggedUser.hasAuthority("ADMIN"))) {
+//			LOGGER.warn("User {} tried to unenroll a student {} from a course and doesn't have DELETE_INSCRIPTION authority [POST]", loggedUser.getDni(), docket);
+//			return new ModelAndView(UNAUTHORIZED);
+//		}
+//		if (errors.hasErrors()) {
+//			LOGGER.warn("User {} could not unenroll docket {} due to {} [POST]", loggedUser.getDni(), docket, errors.getAllErrors());
+//			redirectAttributes.addFlashAttribute("org.springframework.validation.BindingResult.inscriptionForm", errors);
+//			redirectAttributes.addFlashAttribute("inscriptionForm", inscriptionForm);
+//			return new ModelAndView("redirect:/students/" + docket + "/courses");
+//		}
+//
+//		final boolean done = studentService.unenroll(inscriptionForm.getStudentDocket(), inscriptionForm.getCourseId());
+//
+//		if (!done) {
+//			LOGGER.warn("User {} could not unenroll a student {}, Result = {}", loggedUser.getDni(), docket, done);
+//			redirectAttributes.addFlashAttribute("alert", "danger");
+//			redirectAttributes.addFlashAttribute("message", messageSource.getMessage("ERROR_UNKNOWN", null, Locale.getDefault()));
+//		} else {
+//			LOGGER.info("User {} unenrolled student {} successfully", loggedUser.getDni(), docket);
+//			redirectAttributes.addFlashAttribute("alert", "success");
+//			redirectAttributes.addFlashAttribute("message", messageSource.getMessage("unenroll_success",
+//					new Object[] {inscriptionForm.getCourseId(), inscriptionForm.getCourseName()},
+//					Locale.getDefault()));
+//		}
+//
+//		return new ModelAndView("redirect:/students/" + docket + "/courses");
+//
+//	}
+
+	@RequestMapping(value = "/students/{docket}/inscription", method = RequestMethod.GET)
+	public ModelAndView studentInscription(@PathVariable final int docket, final Model model,
+	                                       @ModelAttribute("user") UserSessionDetails loggedUser,
+	                                       @Valid @ModelAttribute("courseFilterForm")
+	                                           final CourseFilterForm courseFilterForm,
+	                                       final BindingResult errors) {
+
+		if (!loggedUser.hasAuthority("ADD_INSCRIPTION")
+				|| (loggedUser.getId() != docket && !loggedUser.hasAuthority("ADMIN"))) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "ADMIN" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)